examples/https-termination/reference-grant.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: ReferenceGrant
+metadata:
+  name: allow-default-to-cafe-secret
+  namespace: certificate
+spec:
+  to:
+  - group: core
```

Comment:
This is the official group name for Secrets, not v1 or empty string?

Reply:
Yes, but it looks like empty string is more common, so I'll change it here.